### DISCUSSION TITLE
movement of the object along the path

### DISCUSSION THREAD

**Discussion by iu0v1** [hajimehoshi/ebiten#1879] [Q&A] [answered]
Hey!
I'm trying my hand at learning Ebiten and stumbled upon a subj issue.
My plan was to describe the vector path and then get the nodes along its way and handle each node individually. But I did not find any methods that would help me achieve this goal.
Maybe I'm missing something? Іs there any idiomatic way to do this? Thanks.

**Comment by hajimehoshi** [comment, marked as answer]
There is no idiomatic way so far, but I'd calculate where to render the image for each frame by myself...

**Reply by iu0v1** [reply]
Oh, gotcha. I'll try to solve it in such a way. Thanks.
